csunplugged/resources/views/modulo_clock.py

```diff
@@ -0,0 +1,87 @@
+"""Module for generating Binary Cards resource."""
+
+from math import pi, sin, cos
+from PIL import Image, ImageDraw, ImageFont
+from utils.retrieve_query_parameter import retrieve_query_parameter
+
+
+def resource_image(request, resource):
+    """Create a image for Binary Cards resource.
+
+    Args:
+        request: HTTP request object.
+        resource: Object of resource data.
+
+    Returns:
+        A list of Pillow image objects.
+    """
+    parameter_options = valid_options()
+    modulo_number = int(retrieve_query_parameter(request, "modulo_number", parameter_options["modulo_number"]))
+    if modulo_number == 2:
+        image_path = "static/img/resources/modulo-clock/modulo-clock-2.png"
+    elif modulo_number == 10:
```

Comment:
Codecov is stating that this line is only partially covered from branching, possibly because it's not an `else`. @ravenmaster001, would you agree?